static/js/settings_account.js

```diff
@@ -696,4 +696,38 @@ export function set_up() {
             $(".privacy-setting-status").expectOne(),
         );
     });
+
+    $("#send_stream_typing_notifications").on("change", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const data = {
+            send_stream_typing_notifications: $("#send_stream_typing_notifications").prop(
+                "checked",
+            ),
+        };
+        settings_ui.do_settings_change(
+            channel.patch,
+            "/json/settings/display",
+            data,
+            $(".privacy-setting-status").expectOne(),
+        );
+    });
+
+    $("#send_private_typing_notifications").on("change", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const data = {
+            send_private_typing_notifications: $("#send_private_typing_notifications").prop(
+                "checked",
+            ),
+        };
+        settings_ui.do_settings_change(
+            channel.patch,
+            "/json/settings/display",
+            data,
+            $(".privacy-setting-status").expectOne(),
+        );
+    });
```

Comment:
May be we can deduplicate some code here, as there are gonna be more setting and `presence_enabled` url will also be changed. Not sure about which will be more readable, but we can try.

Reply:
Yea, that's a good idea. Made that refactor. Does it look right to you?

About the `presence_enabled` setting, we can add it to the `privacy_settings` array I created once we change the URL it relies on.

Reply:
This is also fine, but we can do something similar to what we do in `settings_notifications.js` https://github.com/zulip/zulip/blob/9fec1fb16d98da9e9dbb44a19ea87b562bc82935/static/js/settings_notifications.js#L79

Reply:
Yeah, that's nicer. Changed. But had to add an `if` for `presence_enabled` which would be eliminated on merging the settings endpoints.